### ISSUE TITLE
Does it support multi-softmax as the objective?

### ISSUE THREAD

**Issue by JingYuan0818** [guitargeek/XGBoost-FastForest#1]

**Comment by guitargeek** [comment]
No, it does not support multi-classification, only binary. You would have a use case for that?

**Comment by guitargeek** [comment]
Closed due to inactivity.
